specification/maps/data-plane/Search/preview/1.0/search.json

```diff
@@ -453,7 +454,7 @@
           {
             "name": "geometries",
             "in": "query",
-            "x-ms-client-name": "geometryIds",
+            "x-ms-client-name": "listPolygons",
```

Comment:
Was this from the Python APIView or something else? This _seems_ like a strange name.

Reply:
@tjprescott  I might misunderstood your comments on `async def get_polygon` method. I will change it back.